testapi.pm

```diff
@@ -1929,6 +1929,21 @@ sub save_memory_dump (%nargs) {
     query_isotovideo('backend_save_memory_dump', \%nargs);
 }
 
+=head2 backup_disks
+
+  backup_disks;
+
+
+I<Currently only qemu backend is supported.>
```

Comment:
And maybe we can say what this actually does ;-) like "Save snapshots of the current state of the disks. The VM must be running for this to work"

Reply:
Judging by the code it looks like it might even work when the VM is not running. However, it makes definitely sense to explicitly state in which cases it works and doesn't work.